### ISSUE TITLE
[Bug]: Funky error states @ on read-only form

### ISSUE THREAD

**Issue by mr-michael** [tallycash/extension#557]
### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

Enter an incorrect ETH address and get error messaging inside the extension that I need to supply a properly-formatted eth address

### What did not work?

I get an OS pop-up that I need to put in a valid address ... 

<img width="799" alt="Screen Shot 2021-12-10 at 3 47 54 PM" src="https://user-images.githubusercontent.com/7005061/145639767-9ca41d60-4ed1-40f3-b5ac-09a75a81fb07.png">

And then get an endless spinner ... 

<img width="366" alt="Screen Shot 2021-12-10 at 3 48 01 PM" src="https://user-images.githubusercontent.com/7005061/145639792-d5793ded-4ae2-4a47-af02-b81681314c5c.png">

To end that or resubmit, I have to click 'back' and enter an addy

**Comment by henryboldi** [comment]
Fixed in #545

**Comment by VladUXUI** [comment]
Downloaded the latest Version from Flowdock. This issue seems to be there
 from![image](https://user-images.githubusercontent.com/80846886/145685204-226b2b67-628e-4c0b-948e-ff1e41a7f685.png)

**Comment by Shadowfiend** [comment]
Yep, think this is an issue of having two different address inputs with disparate logic 😬
